MozillaFoundation/foundation.mozilla.org
# Newsletter Signup Error Handling

# Description

There currently is no error message returned to the user in the case that the 3rd party newsletter sign up service is down or does not return 201. This pull request will now display an API error to the user. These API Errors are different and separate from validation errors, and are treated as such.

Link to sample test page: https://foundation-s-11406-news-g7ddcn.mofostaging.net/en/test-the-invalid-newsletter-signup/
Related PRs/issues: #11406

Login: admin
Password: 4@IMcwJ!(GqoL9^Z


# To Test

An invalid signup snippet (non-existant newsletter) has been created and added to the sample test page. Navigate to that link and attempt to sign up. You should now receive an error message.

Repeat for any locale like /fr/ instead of /en/ and verify localization is occurring.

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP-63)
